gradio/networking.py

```diff
@@ -108,36 +114,12 @@ def start_server(
         app: the FastAPI app object
         server: the server object that is a subclass of uvicorn.Server (used to close the server)
     """
-    server_name = server_name or LOCALHOST_NAME
-    # if port is not specified, search for first available port
-    if server_port is None:
-        port = get_first_available_port(

```

Comment:
I haven't removed this method `get_first_available_port` from the code since we use it in reload mode. I do think we need to overhaul reload mode at some point, and then we could remove this method too